source/engineering/abaqus/index.md

```diff
@@ -21,9 +21,8 @@ if you are unfamiliar or need a refresher, then the following courses are recomm
 ## Setup
 
 ```{important}
-Before using Abaqus at the university, you must first register for an Abaqus license
-[here](https://forms.office.com/pages/responsepage.aspx?id=MH_ksn3NTkql2rGM8aQVG0Ep7WjCs19BiAIhW6hXMWlUREtGTFQwS0sySUtGNzZON1MzR1A5TUJKMS4u).
-Please speak to your supervisor to obtain a charge code for the license.
+Before using Abaqus at the university, you must first register for an Abaqus license -  
```

Comment:
```suggestion
Before using Abaqus at the university, you must first register for an Abaqus license -
```

Suggest removing double space after hyphen, as this leads to a line-break in rendered text.